### PR TITLE
add decorator for deprecated Ops

### PR DESCRIPTION
**Pull request by wangchaochaohu**
### PR types

Others
### PR changes

OPs
### Describe

add decorator for deprecated Ops

### CODE REVIEW

**Review by heavengate — DISMISSED** [review]

**Review by TCChenlong — DISMISSED** [review]
LGTM

**Review by heavengate — DISMISSED** [review]

File: python/paddle/fluid/layers/tensor.py [hunk 1]
```diff
@@ -746,6 +747,7 @@ def fill_constant(shape, dtype, value, force_cpu=False, out=None, name=None):
     return out
 
 
+@deprecated(since='1.8.0', update_to="paddle.fluid.fill_constant")
```
Comment (TCChenlong) [inline python/paddle/fluid/layers/tensor.py]: paddle.fill_constant

**Review by TCChenlong — APPROVED** [review]
LGTM

File: python/paddle/fluid/layers/tensor.py (hunk 1 above)
Comment (wangchaochaohu) [reply python/paddle/fluid/layers/tensor.py]: Done

**Review by lanxianghit — APPROVED** [review]

**Pull request merged**